### ISSUE TITLE
Feature: specify cront start time

### ISSUE THREAD

**Issue by Bilge** [travis-ci/travis-ci#8705]
It seems cron jobs recur at the time at the same time of day which the job was enabled. If one is to enable cron jobs for many projects at once, this will cause a concurrency bottleneck between the projects. If one wants to start a job at a specific time of day, one has to be awake at that time of day and enable the job at exactly that time; not very practical if one wants to schedule a job for 4am.

It seems to me there should be an option to specify the time of day at which a cron job starts.

**Comment by Bilge** [comment]
Please don't close this issue. It's very important to me, and seemingly to many others. Even if you know the trick to get your job to start at the desired time, it's overall useless because the start time drifts gradually (by several hours over a period of 6 months).

@BanzaiMan Is this still on the radar?

**Comment by georgetulakyan** [comment]
This is still an issue. Why has this been closed. You expect developers to sit around and wait until for the time they want the cron jobs of the projects come up. How about if they want the job to run at 2 am? Seriousness, fix this. This is an issue that a beta version would have.

**Comment by Bilge** [comment]
@georgetulakyan JUST USE ANOTHER CI LOOOOOOOOOOOOOOL :joy:

**Comment by georgetulakyan** [comment]
Why? We are already invested time and resources using this one.

**Comment by georgetulakyan** [comment]
@Bilge , what are you using?

**Comment by Bilge** [comment]
Azure :ok_hand:

**Comment by georgetulakyan** [comment]
Azure Pipelines?

**Comment by ulope** [comment]
At this point it’s become clear that Travis is not interested in or capable of improving.

We’ve moved to circle now and while nothing is perfect are very happy with it (also our build now takes a 5th if the time).

**Comment by georgetulakyan** [comment]
@ulope you paying or using the free plan?

**Comment by ulope** [comment]
@georgetulakyan paying, but payed for Travis as well.

**Comment by JimLynchCodes** [comment]
Hey all! I would just like to express my similar feelings of rage towards this half-assed "cron jobs" feature that does not even allow one to set the time of day. DERP?! 

![jackie-what](https://user-images.githubusercontent.com/5354163/82830348-793a7500-9e83-11ea-946a-f34890529b3a.jpeg)

**Comment by eromano** [comment]
I agree, guys, why you don't keep iterate on this half, made functionality ??? are you waiting for people to get too frustrated and start to use other products? Should I wake up at 2 o clock in the night to set cron at that time?????.... 

> ![Gordon](https://media.giphy.com/media/okFDq1MPeU2UE/giphy.gif)

**Comment by eromano** [comment]
and you keep closing all those requests...

**Comment by eromano** [comment]
2022 still not implemented... 2 years later

<img src="https://media1.giphy.com/media/d31w24psGYeekCZy/giphy.gif"/>

**Comment by Bilge** [comment]
It's [implemented here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule). Travis is dead; stop clinging to the past.
